### PR TITLE
Trending

### PR DESCRIPTION
**Pull request by PauloGabrieel**
I created the flush to render the trending on the timeline with the data coming from the api.
-- when clicking on the hashtag and redirected to the route /hashtag/:hashtag.

-- :hashtag is the name of the tag without the #.
-- the hashtag route was not created in this feature.

### CODE REVIEW

**Review by V1T0R-CM — APPROVED** [review]

File: src/components/PublicationForm.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ export default function PublicationForm(){
     const [postLink, setPostLink] = useState("");
     const [content, setContent] = useState(""); 
     const [actionDisabled, setActionDisabled] = useState(false);
-        
-    const navigate = useNavigate()
+    let navigate = useNavigate();
```
Comment (GabrielCmrg) [inline src/components/PublicationForm.js]: why change this to let instead of const?

File: src/services/api.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import axios from "axios";
 
-const { REACT_APP_API_URL } = process.env;
+const  {REACT_APP_API_URL}  = process.env;
+console.log(REACT_APP_API_URL);
```
Comment (GabrielCmrg) [inline src/services/api.js]: this can be removed

**Review by GabrielCmrg — APPROVED** [review]

File: src/pages/Timeline.js [hunk 3]
```diff
@@ -1,19 +1,112 @@
...
+            return (
+                <TextContainer>
+                    <h2>There are no posts yet</h2>
+                </TextContainer>
+            );
+        }else{
+            return(
+                posts.map((item,index)=>(
+                    <Publication  
+                        key={index}
```
Comment (GabrielCmrg) [inline src/pages/Timeline.js]: Please change this to item.id because it will be needed for deleting and editing.

**Review by GabrielCmrg — CHANGES_REQUESTED** [review]

File: src/components/PublicationForm.js (hunk 1 above)
Comment (PauloGabrieel) [reply src/components/PublicationForm.js]: I was testing something, but I already removed it.

File: src/services/api.js (hunk 2 above)
Comment (PauloGabrieel) [reply src/services/api.js]: Done.

File: src/pages/Timeline.js (hunk 3 above)
Comment (PauloGabrieel) [reply src/pages/Timeline.js]: Done.

**Review by GabrielCmrg — APPROVED** [review]

**Pull request merged**